WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCard.swift

```diff
@@ -106,27 +119,31 @@ private extension ReviewsDashboardCard {
         }
     }
 
-    func ReviewRow(for review: ProductReview, isLastItem: Bool) -> some View {
+    func ReviewRow(for viewModel: ReviewViewModel, isLastItem: Bool) -> some View {
```

Comment:
```suggestion
    func reviewRow(for viewModel: ReviewViewModel, isLastItem: Bool) -> some View {
```

Reply:
Ah, nice! Subconsciously used Jetpack Compose habit here with capitalized name 😅 

Fixed in https://github.com/woocommerce/woocommerce-ios/pull/12752/commits/4fa99a1b9039bb6d0fdffa3af80b2be9a8950bb1 as it requires a change elsewhere too.